src/i18n/strings/en_EN.json

```diff
@@ -2423,7 +2424,6 @@
     "Failed to send logs: ": "Failed to send logs: ",
     "Preparing to download logs": "Preparing to download logs",
     "Reminder: Your browser is unsupported, so your experience may be unpredictable.": "Reminder: Your browser is unsupported, so your experience may be unpredictable.",
-    "Debug logs contain application usage data including your username, the IDs or aliases of the rooms or groups you have visited, which UI elements you last interacted with, and the usernames of other users. They do not contain messages.": "Debug logs contain application usage data including your username, the IDs or aliases of the rooms or groups you have visited, which UI elements you last interacted with, and the usernames of other users. They do not contain messages.",
```

Comment:
This came from here: https://github.com/matrix-org/matrix-react-sdk/blob/86329012f37e88ef052c84c6e59639738d7a81b0/src/components/views/dialogs/BugReportDialog.tsx#L206-L209